.yamlfmt

```diff
@@ -0,0 +1,4 @@
+formatter:
```

Comment:
Should we move this configuration file out of the root directory to reduce initial clutter?

It's possible to specify the file location via https://github.com/google/yamlfmt/blob/main/docs/command-usage.md#configuration-flags 

It could sit under `/tools` or `/tests` instead perhaps.  Just an idea given we have been [discussing tidying root directory](https://github.com/etcd-io/etcd/issues/15778).

In hindsight we could actually do the same for `yamllint` https://yamllint.readthedocs.io/en/stable/configuration.html.

Reply:
Thank you for your suggestion. Using parameters can indeed reduce the content of the root directory.

There are several reasons why I use configuration files:

* These configuration files are hidden by default, and most people will not see them directly;
* The names of these configuration files are very intuitive, and their functions can be understood through their names.


The difference from https://github.com/etcd-io/etcd/issues/15778 is that we expect `yamllint` and `yamlfmt` to be triggered through `make`, so there is no need to touch these configuration files.
But the Procfile file is included in the documentation, and people need to understand its function and significance.



If modifications are needed, I can make adjustments. `yamlfmt` passes parameters through flags, and the configuration file for `yamllint` is located in either the `tools` or `scripts` directory.

Reply:
Thanks @tao12345666333, my preference would be to:

1. Move `.yamllint` and `.yamlfmt` to `tools/...`
2. Update `Makefile` as follows:

```diff
verify-yamllint:
-	yamllint .
+      yamllint --config-file tools/.yamllint .
```

```diff
fix-yamllint:
ifeq (, $(shell which yamlfmt))
	$(shell go install github.com/google/yamlfmt/cmd/yamlfmt@latest)
endif
-	yamlfmt .
+      yamlfmt -conf tools/.yamllint .

```

Reply:
Addressed!

Reply:
Agree with the proposed direction. @jmhbnz Can you file an issue to consider moving other dotfiles?

Reply:
FYI there is also `hack` directory borrowed from kubernetes that serves similar purpose.

Reply:
Don't like the idea of putting the config files under `hack`.  

https://github.com/etcd-io/etcd/issues/15987#issuecomment-1571393182